partcad-cli/src/partcad_cli/click/commands/add/sketch.py

```
@@ -51,5 +51,13 @@ def cli(ctx, desc, kind, provider, path):
             config["desc"] = desc
         if provider:
             config["provider"] = provider
+            kind_ext = {
+                "ai-cadquery": "py",
+                "ai-openscad": "scad",
+            }
+            if path.lower().endswith((".%s" % kind_ext[kind]).lower()):
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)